themes/default/content/docs/guides/organizing-projects-stacks/_index.md

```diff
@@ -177,3 +275,101 @@ These component resources can be packaged up and stored alongside all of your ot
 ### Other examples
 
 See also the use of multiple projects and stacks in [Crosswalk for Kubernetes]({{< relref "/docs/guides/crosswalk/kubernetes" >}}), which contains a tutorial, reference architecture, and collection of prod-first code examples that demonstrate industry best-practices for using Kubernetes in contexts where an organization of people must ship production applications.
+
+## Organizing your project code
+
+Within your Pulumi project, there are good practices to consider to help keep your code organized, maintainable, and understandable.
+
+{{< chooser language "typescript,go" / >}}
+
+{{% choosable language typescript %}}
+Organize your code in a way that makes it easy to understand and maintain. One way to do this in Typescript is to break out your code into separate files, and then import them into your main file. In this example, the entrypoint for our Pulumi program is `index.ts`, but we use the `utils.ts` file for supporting functions.
+
+```typescript
+// index.ts
+import * as util from "./util";
+...
+const forwarderHandle = util.forwardPrometheusService(p8sService, p8sDeployment, {
+    localPort,
+});
+```
+
+```typescript
+// utils.ts
+import * as k8s from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
+
+export function forwardPrometheusService(
```

Comment:
Do you think this function is more complex that it needs to be to get the point across? I wonder if something super-simple wouldn't be better -- like a string formatter or something that's more "util"-y. There's a bunch of stuff here that, while awesome, probably isn't super relevant to what you're trying to convey. What do you think?

Reply:
Yep, that’s really a good idea. Otherwise this does start to also drift a bit into component resources as well which could confuse the point. And it’ll be much easier to illustrate cross-languages with something like this! Thanks for bumping me into the better direction.